content/docs/what-is-kudo.md

```diff
@@ -0,0 +1,117 @@
+
+A _KUDO Manager_ is a set of k8s controllers that understand KUDO operators and know how to execute plans.
+
+A _Plan_ is the operator's main workflow unit. A plan specifies a series of steps that will apply or delete k8s resources to the cluster in the defined order.
+
+**In a nutshell**: A user will take an _Operator Package_ from the _Repository_, submit it to the _KUDO Manager_ which will then execute operator _Plans_ either automatically or on-demand.
+
+## Operator PLans
+
+An operator typically consists of several plans. Think of them as runbooks written in a structured way that can be executed by KUDO. Plans are made up of phases, and phases have one or more steps. Every operator must contain at least a `deploy` plan which is the default plan to deploy an application to the cluster. For more complex applications, you would want to define a plan for backup and restore or upgrade. Phases and steps can be executed either serially or in parallel. The default execution strategy, if none is specified, is _serial_.
+
+```text
++------------------------------------+
+|  Plan                              |
+|                                    |
+|    +-----------------------+       |
+|    |  Phase                +-+     |
+|    |     +------------+    +---+   |
+|    |     |  Step      ++   | | |   |
+|    |     |            |-+  | | |   |
+|    |     +--------------|  | | |   |
+|    |      +-------------|  | | |   |
+|    |        +-----------+  | | |   |
+|    |                       | | |   |
+|    +-----------------------+ | |   |
+|      +-----------------------+ |   |
+|        +-----------------------+   |
+|                                    |
+|                                    |
++------------------------------------+
+```
+
+Here is an example of a `deploy` plan for some abstract data service, where different node types (_bootstrap, master, agent_) has to be started in a specific serial order:
+
+```text
+plan: deploy
+├── phase: bootstrap-node
+│   └── step: start-bootstrap-node
+├── phase: master-nodes
+│   └── step: start-master-nodes
+└── phase: agent-nodes
+│   └── step: start-agent-nodes
+└── phase: deploy-cleanup
+    └── step: remove-bootstrap-node
+```
+
+After the successful deployment, the `bootstrap` node is removed. Each step has to be finished successfully e.g. corresponding nodes are running, healthy (and exposed to the network), before KUDO continues with the execution of the next phase. Learn more about plans by [writing you first operator](???) and take a look at the [operator documentation](???)
+
+## Operator Parameters
+
+Operators can be customized using proved parameters. Where KUDO differs from a lot fo similar tools: _operator parameters are tied to plans, so that when a parameter changes a plan would be automatically triggered._ Here are two typical parameters for the number of nodes and memory that each node container is using:
```

Comment:
```suggestion
Operators can be customized using provided parameters. Where KUDO differs from a lot of similar tools: _operator parameters are tied to plans, so that when a parameter changes a plan would be automatically triggered._ Here are two typical parameters for the number of nodes and memory that each node container is using:
```

I assume `provided` is meant?